staging/src/k8s.io/apiextensions-apiserver/pkg/utils/helpers.go

```diff
@@ -14,17 +14,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apiextensions
+package utils
```

Comment:
There are helpers for the api. Don't like to move them into some generic pkg/utils package. Why do we need that?

Reply:
Another reason for that: the helpers are about internal types. They shouldn't be used outside of the apiserver itself.